### PR TITLE
fix: repair broken gitignore file

### PR DESCRIPTION
**Pull request by elektrovert**
I've updated the .gitignore file, you can see details on the individual commits.  
I think you can set the repository here to always rebase when merging, like we set in git on your pc.   
You should do that too, merging with rebase is the best way to do it, otherwise things can get really messy.  
When you're bored some day, check out git conventional commits, it's a standard for how you should write git messages.
https://www.conventionalcommits.org/en/v1.0.0/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,3 @@
-<<<<<<< HEAD
```
Comment (elektrovert) [inline .gitignore:1]: This is part of a git merge request, HEAD means the current position of your git repository; you're supposed to examine the code between here and the next marker and compare to the remaining part.  
Then pick one as the correct version (or a mix of the two) and delete the rest and recommit.  
Your IDE could handle this for you too. In intellij this looks really nice, there's a window in the git menu called "resolve conflicts" (I think) that will guide you though the process.

File: .gitignore [hunk 2]
```diff
@@ -50,7 +48,94 @@ replay_pid*
 
 # Support for Project snippet scope
 
-=======
```
Comment (elektrovert) [inline .gitignore:53]: This is the second marker, everything below this is another version of the stuff above.

File: .gitignore [hunk 3]
```diff
@@ -50,7 +48,94 @@ replay_pid*
...
+# Created by https://www.toptal.com/developers/gitignore/api/intellij
+# Edit at https://www.toptal.com/developers/gitignore?templates=intellij
 
-# Created by https://www.toptal.com/developers/gitignore/api/visualstudiocode,java
-# Edit at https://www.toptal.com/developers/gitignore?templates=visualstudiocode,java
```
Comment (elektrovert) [inline .gitignore:56]: There is usually another git marker here to say which commit the code above relates to, but it wasn't in this file, maybe you deleted it already?

Comment (charb29) [reply .gitignore:56]: I haven't touched the Intelli J .gitignore yet. I'll look through it

**Pull request merged**